src/demo-app/stepper/stepper-demo.html

```diff
@@ -1,3 +1,46 @@
+<h2>Linear Vertical Stepper Demo</h2>
+<form [formGroup]="formGroup">
+  <md-vertical-stepper formArrayName="formArray" linear>
```

Comment:
can we do `[linear]="isLinear"` and add a checkbox to toggle it, that way its easy to test